model/alertrecord/alert_bookkeeping.go

```diff
@@ -127,6 +128,13 @@ func ByFirstFailureInTaskType(subscriptionID, versionId, taskName string) db.Q {
 	return db.Query(q).Limit(1)
 }
 
+func ByFirstRegressionInVersion(subscriptionID, versionId string) db.Q {
```

Comment:
nit, but our trend is to publically expose Find... functions, so instead of exposing the query, just can you just make this FindByFirstRegressionInVersion function that returns the alert record. Also make the above constant private?

Reply:
Done. The constant still needs to be public because we create alert records with it